### PR TITLE
Subset maker fixes

### PR DESCRIPTION
**Pull request by alexcouper**
Changes to the API mean that this change is needed.

### CODE REVIEW

**Review by sindrig — APPROVED** [review]
LGTM

File: cmd/create_opportunity.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func init() {
 	createCmd.AddCommand(opportunityCmd)
 	opportunityCmd.Flags().StringVar(&uid, "uid", "", "Unique identifier for the opportunity")
 	opportunityCmd.Flags().StringVar(&name, "name", "", "Name of the opportunity")
-	opportunityCmd.Flags().StringVar(&score, "score", string(openapi.OpportunityScoreUnknown), "Risk score of the opportunity (critical, high, medium, low, info, none, unknown)")
```
Comment (sindrig) [inline cmd/create_opportunity.go:94]: Would it make more sense to keep this as a `StringVar` and translate the strings to ints before sending to aftra?

Comment (alexcouper) [reply cmd/create_opportunity.go:94]: I'm asking @samuelhafsteinsson about the API change [here](https://github.com/syndis-software/aftra/pull/3404/files#r1506184114)

Comment (alexcouper) [reply cmd/create_opportunity.go:94]: We're going to leave this for now.

**Pull request merged**